src/main/java/quickndirty/minisurveymonkey/TextQuestion.java

```diff
@@ -1,10 +1,14 @@
 package quickndirty.minisurveymonkey;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class TextQuestion extends Question {
 
 	public TextQuestion(String Prompt) {
 		type = QuestionType.TEXT;
 		this.prompt = Prompt;
+		responses = new ArrayList<Response>();
```

Comment:
Would it make sense to do this in `Question`? I think would apply to any subclasses.

Reply:
Yeah I'll make that change